### PR TITLE
Issue #2311: Basis: fix email field label and asterisks in contact form

### PR DESCRIPTION
**Pull request by klonos**
Fixes https://github.com/backdrop/backdrop-issues/issues/2311

### CODE REVIEW

**Comment by wesruv** [comment]
I'm good with everything except removing the underline... for accessibility sake I think we should keep an underline on all elements that are using `<abbr>` element.

File: core/themes/basis/css/component/backdrop-form.css [hunk 1]
```diff
@@ -8,15 +8,26 @@
...
+.form-type-textarea label,
+.form-type-textfield label {
   display: block;
   margin: 0 0 0.25em;
 }
 
+abbr.form-required,
+abbr.tabledrag-changed,
+abbr.ajax-changed {
+    border-bottom: none;
```
Comment (wesruv) [inline core/themes/basis/css/component/backdrop-form.css:24]: This needs to be removed

**Review by wesruv — CHANGES_REQUESTED** [review]

**Comment by wesruv** [comment]
Thanks for tackling this @klonos !

**Comment by quicksketch** [comment]
Merged into 1.x and 1.5.x.

**Pull request merged**